binance/client.py

```diff
@@ -1932,7 +1932,7 @@ def get_account_status(self, **params):
 
         """
         res = self._request_withdraw_api('get', 'accountStatus.html', True, data=params)
-        if not res.get('success'):
```

Comment:
I think you should use this code `if not res.get('success'):` everywere...

Reply:
Fair point. I think I've had older version where it just called not res['success'] directly. 
Will stick with that.